server/boards/app/files.go

```diff
@@ -175,3 +190,165 @@ func (a *App) MoveFile(channelID, teamID, boardID, filename string) error {
 	}
 	return nil
 }
+
+func (a *App) CopyAndUpdateCardFiles(boardID, userID string, blocks []*model.Block, asTemplate bool) error {
+	if err := a.CopyCardFiles(boardID, blocks, asTemplate); err != nil {
+		a.logger.Error("Could not copy files while duplicating board", mlog.String("BoardID", boardID), mlog.Err(err))
+	}
+
+	// blocks now has updated file ids for any blocks containing files.  We need to update the database for them.
+	blockIDs := make([]string, 0)
+	blockPatches := make([]model.BlockPatch, 0)
+	for _, block := range blocks {
+		fieldName := ""
+		if block.Type == model.TypeImage {
+			fieldName = "fileId"
+		} else if block.Type == model.TypeAttachment {
+			fieldName = "attachmentId"
```

Comment:
I think we should standardize on `fileId` for both types.  I know we have to continue reading both fields for now, but we can starting writing a standardized name.

Reply:
Done, required more changes, but good call.